assets/js/components/higherorder/withdata.js

```diff
@@ -145,14 +145,15 @@ const withData = (
 			 *
 			 * @param {Object} returnedData The data returned from the API.
 			 * @param {string} datapoint    The datapoint name resolved by the API.
-			 * @param {string} identifier   The object name, typically the module name eg 'analytics.
+			 * @param {Object} requestData  The data object for the request.
 			 */
-			const handleReturnedData = ( returnedData, datapoint, identifier ) => {
+			const handleReturnedData = ( returnedData, datapoint, requestData ) => {
 				// If available, `handleDataError` will be called for errors (with a string) and empty data.
 				const {
 					handleDataError,
 					handleDataSuccess,
 				} = this.props;
+				const { identifier } = requestData;
```

Comment:
While we're at it, should we also remove the `datapoint` parameter and here get it from `requestData`?

Reply:
Done!